cmd/atc/internal/testing/flights/crossnamespace/main.go

```diff
@@ -12,7 +13,7 @@ import (
 
 func main() {
 	// This flight needs cross-namespace to be set in order to work and serves to test that feature.
-	json.NewEncoder(os.Stdout).Encode(flight.Resources{
+	if err := json.NewEncoder(os.Stdout).Encode(flight.Resources{
```

Comment:
this is just testing. If it fails it will get caught in the tests.
Easier to just ignore the error explicitly:

_ = json.NewEncoder(...).Encode(...)